Website/DesktopModules/Hotcakes/Core/Admin/Parts/PaymentMethods/PaypalExpress/App_LocalResources/Edit.ascx.resx

```diff
@@ -117,14 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="APIPassword.Text" xml:space="preserve">
-    <value>API Password</value>
+  
+  <data name="APIClientId.Text" xml:space="preserve">
+    <value>API Client Id</value>
```

Comment:
Is this the same text that PayPal uses in its own documentation? If not, we should make it the same phrase to reduce confusion.

Reply:
It is like this but without the API word. Should I removed it?